_includes/sponsor-grid.html

```diff
@@ -0,0 +1,43 @@
+{%- assign sponsor500 = site.data.sponsor-list | where: 'category', '500' %}
+{%- if sponsor500.size > 0 %}
+<h4>The following people or companies are sponsoring us for $500 or more per month:</h4>
+
+<div id="sponsors" class="sponsors500">
+  <ul>
+    {%- for sponsor in sponsor500 %}
+      <li>
+        {%- if sponsor.url != '' %}
+          <a href="{{ sponsor.url }}" title="{{ sponsor.name }}">
+        {% endif -%}
+          {% include sponsor-logos/{{ sponsor.logo }} %}
```

Comment:
If you are afraid of there not being svg versions of the images, you could instead place the logos in `assets/images` and `<img src="">` them here

Reply:
I think we can just require them to provide svgs, or offer to convert them with the caveat that it may look slightly off.